calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
+
+exports.create_calculator = function (operator) {
+  const numbers = [];
+
+  return {
+    add_number: function (number) {
+      numbers.push(number);
+    },
+    calculate: function () {
+      // Return zero if there are no numbers to calculate
```

Comment:
This comment is a lie! It does not return zero, it throws an error!

Reply:
In general we should be highly skeptical of all code comments since there's nothing programmatically enforcing their correctness